packages/cli/src/commands/projects.ts

```diff
@@ -103,16 +117,144 @@ export default async (client: Client) => {
 
 async function run({
   client,
-  contextName,
+  scope,
 }: {
   client: Client;
-  contextName: string;
+  scope: {
+    contextName: string;
+    team: Team | null;
+    user: User;
+  };
 }) {
   const { output } = client;
+  const { contextName, team } = scope;
   const args = argv._.slice(1);
 
   const start = Date.now();
 
+  if (subcommand === 'connect') {
+    const yes = Boolean(argv['--yes']);
+    if (args.length !== 0) {
+      console.error(
+        error(
+          `Invalid number of arguments. Usage: ${chalk.cyan(
+            `${getCommandName('projects connect')}`
+          )}`
+        )
+      );
+      return exit(2);
+    }
+
+    let paths = [process.cwd()];
+
+    const validate = await validatePaths(client, paths);
+    if (!validate.valid) {
+      return validate.exitCode;
+    }
+    const { path } = validate;
+
+    const linkedProject = await ensureLink(
+      'projects connect',
+      client,
+      path,
+      yes
+    );
+    if (typeof linkedProject === 'number') {
+      return linkedProject;
+    }
+
+    const { project, org } = linkedProject;
+    const gitProviderLink = project.link;
+
+    client.config.currentTeam = org.type === 'team' ? org.id : undefined;
+
+    // get project from .git
+    const gitConfigPath = join(path, '.git/config');
+    const gitConfig = await parseGitConfig(gitConfigPath, output);
+    if (!gitConfig) {
+      output.error(
+        `No local git repo found. Run ${chalk.cyan(
+          '`git clone <url>`'
+        )} to clone a remote Git repository first.`
+      );
+      return 1;
+    }
+    const remoteUrl = pluckRemoteUrl(gitConfig);
+    if (!remoteUrl) {
+      output.error(
+        `No remote origin url found in your Git config. Make sure you've connected your local Git repo to a Git provider first.`
+      );
+      return 1;
+    }
+    const parsedUrl = parseRepoUrl(remoteUrl);
+    if (!parsedUrl) {
+      output.error(
+        `Can't parse Git repo data from the following remote url in your Git config: ${remoteUrl}`
```

Comment:
```suggestion
        `Failed to parse Git repo data from the following remote URL in your Git config: ${remoteUrl}`
```

Reply:
Might also want to add formatting on the `remoteUrl` i.e. from `util/output/link.ts`.